common/app/views/fragments/amp/stylesheets/tones.scala.html

```diff
@@ -21,7 +21,8 @@
     color: #fff;
 }
 
-.tonal--tone-comment .tonal__standfirst {
+.tonal--tone-comment .tonal__standfirst,
```

Comment:
The AMP spec mandates non-nested selectors, ie strict BEM. Not sure that they actually throw validation errors for it though. Have you tried validating it?